central/notifiers/email/email.go

```diff
@@ -592,9 +592,9 @@ func PlainTextAlert(alert *storage.Alert, uiEndpoint string, mitreStore mitreDS.
 	return notifiers.FormatAlert(alert, alertLink, funcMap, mitreStore)
 }
 
-func BuildReportMessage(recipients []string, from, subject, messageText string, zippedReportData *bytes.Buffer) Message {
+func BuildReportMessage(recipients []string, from, subject, messageText string, zippedReportData *bytes.Buffer, reportName string) Message {
 	if subject == "" {
-		subject = fmt.Sprintf("%s Image Vulnerability Report for %s", branding.GetProductNameShort(), time.Now().Format("02-January-2006"))
+		subject = fmt.Sprintf("%s %s for %s", branding.GetProductNameShort(), reportName, time.Now().Format("02-January-2006"))
```

Comment:
How about `%s report '%s' for %s` format? 
So it would be like "RHACS report 'Test VM report' for 16-September-2024"

Reply:
I like the clarity of your suggestion, but I found several references that say quote marks in subject lines are a trigger for spam filters, so I'm going to omit the quote mark part.